.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+version: 2.1
+executors:
+  default:
+    working_directory: ~/mattermost-webapp
+    docker:
+      - image: mattermost/mattermost-build-webapp:oct-2-2018
+
+jobs:
+  install:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - dependencies-{{ checksum "package-lock.json" }}
+      - run: |
+          npm ci
+          cd node_modules/mattermost-redux && npm i && npm run build
```

Comment:
💯

Reply:
we definitely need to fix this soon. The ticket belongs to platform https://mattermost.atlassian.net/browse/MM-13942 but is assigned as HW, perhaps this is something that we should do directly